microprofile/fault-tolerance-smallrye/executor/pom.xml

```diff
@@ -25,12 +25,13 @@
 
     <parent>
         <groupId>org.wildfly</groupId>
-        <artifactId>wildfly-microprofile-fault-tolerance-smallrye</artifactId>
+        <artifactId>wildfly-parent</artifactId>
```

Comment:
This is wrong, why this change? This breaks `mvn clean install -rf :wildfly-microprofile-fault-tolerance-smallrye`.

Reply:
We might be able to get rid of it.  It's a long story.

Reply:
OK I trust you! Just wanted to point it out in case it was unintentional.